src/05-excitation-transfer.md

```diff
@@ -0,0 +1,762 @@
+    df_odd.iloc[i,1:] = evals 
+```
+
+```python
+# EVEN PARITY
+
+# make the operators
+two_state_1, two_state_2, bosons, interaction_1, interaction_2, number, nmm_list = make_operators(
+    max_bosons=6, parity=1)
+
+# prepare data structure for the energy level scan
+df_even = make_df_for_energy_scan("$\Delta E$", -4, 4, 201, two_state_1.shape[0])
+
+# fill the data structure with eigevalues of the Hamiltonian i.e. the energy levels
+for i, row in df_even.iterrows():
+    H =  row["$\Delta E$"]*two_state_1+ row["$\Delta E$"]*two_state_2 + 1*bosons 
+    evals, ekets = H.eigenstates()
+    df_even.iloc[i,1:] = evals 
+```
+
+```python
+fig, axes = plt.subplots(nrows=1, ncols=2, figsize=(15,6), sharey=True)
+
+fig.suptitle("Energy levels for $H = \Delta E /2 (\sigma_{z1} + \sigma_{z2}) + \hbar\omega(a^{{\dagger}}a +1/2) $  ($\omega=1$)")
+
+
+df_odd.plot(x="$\Delta E$",ylim=[-0.5,5.5],legend=False, 
+        title="Odd parity (Fig 5)",  ax=axes[0]);
+
+df_even.plot(x="$\Delta E$",ylim=[-0.5,5.5],legend=False, 
+        title="Even parity   (Fig 6)",  ax=axes[1]);
+
+axes[0].set_ylabel("Energy");
+
+
+```
+
+What can we say about th energy levels in Figs 5 and 6?
+
+Compared to [Fig 1 of tutorial 4](https://nbviewer.jupyter.org/github/project-ida/two-state-quantum-systems/blob/master/04-spin-boson-model.ipynb#4.2.1---Spin-boson-landscape-$U=0$), we have some similarities and some differences:
+- Similarity - levels that trend up and down with changing $\Delta E$. These correspond to states like $|n,+,+\rangle$ and $|n,-,-\rangle$. For example, in Fig 6 for small positive $\Delta E$ (referring to blue squares in Fig 4 for even parity states) we have:
+  - $|0,-,-\rangle$ - blue
+  - $|0,+,+\rangle$ - orange 
+  - $|2,-,-\rangle$ - purple  
+  - $|2,+,+\rangle$ - brown 
+  - $|4,-,-\rangle$ - yellow 
+  - $|4,+,+\rangle$ - light-blue 
+- Difference - horizontal levels. These correspond to combinations of $|n,+,-\rangle$ and $|n,-,+\rangle$ which give is no overall "excitation" i.e. equal amounts of + and -. This kind of combination of basis states is often referred to as an [entangled state](https://en.wikipedia.org/wiki/Quantum_entanglement#Pure_states). The subject of quantum entanglement deserves at least a whole tutorial, so we'll come back to it another time.
+
+Thinking more carefully about these unfamiliar horizontal levels, there are 2 ways we can combine $|n,+,-\rangle$ and $|n,-,+\rangle$ together:
+- $|n,+,-\rangle + |n,-,+\rangle$ 
+- $|n,+,-\rangle  - |n,-,+\rangle$ 
+
+These combinations are reminiscent of the "in phase" and "out of phase" states that we described [back in tutorial 1](https://nbviewer.jupyter.org/github/project-ida/two-state-quantum-systems/blob/master/01-an-isolated-two-state-system.ipynb#1.2-Coupling-between-two-states-of-the-same-energy).  Because there are two combinations, we can expect that each horizontal line in Fig 6 is in fact 2 horizontal lines on top of each other, i.e:
```

Comment:
maybe we could plot the top one in broken style, although I guess that here it is complicated

Reply:
Yeah, that's tricky. I've modified the text to refer to the positions of the lines on the graphs as opposed to the colours.